tox.ini

```diff
@@ -28,6 +28,7 @@ commands =
 [testenv:typing]
 deps =
     mypy
+    {[testenv]deps}
```

Comment:
not familiar with this syntax, could you please digest it for me?

Reply:
This causes all `deps` listed in the `[testenv]` environment (i.e., all of our test dependencies) to be included in the `deps` for the `typing` environment.  This is necessary for mypy to be able to type-check test code that uses dependencies like `pytest`.